tests/Console/ImportNasspGroupsCommandTest.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace Tests\Console;
+
+use Tests\TestCase;
+use Rogue\Models\Group;
+use Rogue\Models\GroupType;
+
+class ImportNasspGroupsCommandTest extends TestCase
+{
+    public function testImportingGroups()
+    {
+        $this->artisan('rogue:nassp-groups-import', ['path' => 'tests/Console/example-nassp-groups.csv']);
+
+        // Four group types should have been created.
+        $this->assertTrue(GroupType::all()->count() == 4);
```

Comment:
Small nitpick but can we switch to using `===` on the asserts to check against the count?